cranelift/filetests/src/runner.rs

```diff
@@ -13,10 +13,10 @@ use std::path::{Path, PathBuf};
 use std::time;
 
 /// Timeout in seconds when we're not making progress.
-const TIMEOUT_PANIC: usize = 60;
+const TIMEOUT_PANIC: usize = 600;
```

Comment:
Were these changes prompted by CI issues?

Reply:
Oops, left those in by accident, sorry.